package.json

```diff
@@ -31,8 +28,8 @@
     "tslib": "2.7.0"
   },
   "devDependencies": {
+    "@types/react": "^18.3.11",
     "@types/node": "^22.7.0",
-    "@types/react": "^18.3.9",
     "@types/react-dom": "^18.3.0",
     "eslint": "^9.11.1",
     "eslint": "^9.11.1",
```

Comment:
**Suggestion:** Verify that the duplicate entry of `eslint` in `devDependencies` is intentional or remove the duplicate to clean up the package.json. [maintainability, importance: 8]
```suggestion
    "eslint": "^9.11.1",
```